.github/workflows/ci.yml

```diff
@@ -10,6 +10,19 @@ on:
     - cron:  '0 3 * * *' # daily, at 3am
 
 jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
+
+      - run: yarn install
```

Comment:
should we not cache as in https://github.com/simplabs/ember-cookies/pull/477/files#diff-e9f950f17198d3d5e3122a44230a09b9R56 ?

Reply:
you might want to have a look at the other PR 😅

Reply:
Yeah, just saw that afterwards :)